lib/apachetest.pm

```diff
@@ -100,13 +120,90 @@ sub setup_apache2 {
     # Create a test html page
     assert_script_run 'echo "<html><h2>Hello Linux</h2></html>" > /srv/www/htdocs/hello.html';
 
+    # create a test php page
+    assert_script_run qq{echo -e "<?php\nphpinfo()\n?>" > /srv/www/htdocs/index.php};
+
     # Verify apache+ssl works
     if ($mode =~ m/SSL|NSS/) {
         validate_script_output 'curl -k https://localhost/hello.html', sub { m/Hello Linux/ };
     }
     else {
         validate_script_output 'curl http://localhost/hello.html', sub { m/Hello Linux/ };
     }
+
+    if ($mode =~ /PHP5|PHP7/) {
+        assert_script_run "curl --no-buffer http://localhost/index.php | grep \"\$(uname -s -n -r -v -m)\"";
+    }
+}
+sub setup_pgsqldb {
+    # without changing current working directory we get:
+    # 'could not change directory to "/root": Permission denied'
+    assert_script_run 'pushd /tmp';
+
+    # requires running postgresql94 server
+    # test basic functionality - require postgresql94
+    assert_script_run "sudo -u postgres createdb openQAdb";
```

Comment:
What if we will extract this to *.sql file which we can put into data directory ? I mean all CREATE/INSERT/SELECT lines too ?